scripts/scoring_test.py

```diff
@@ -0,0 +1,29 @@
+import requests
+from bs4 import BeautifulSoup
+import numpy as np
+
+DEBUG_URL = 'http://127.0.0.1:5000'
```

Comment:
I think this is defined in script_config.yml ; if not, add it there and pass it as elsewhere.